java/ci/build-in-docker.sh

```diff
@@ -61,7 +61,7 @@ cmake .. -G"${CMAKE_GENERATOR}" \
          -DCUDF_USE_ARROW_STATIC=ON \
          -DCUDF_ENABLE_ARROW_S3=OFF \
          -DBUILD_TESTS=$BUILD_CPP_TESTS \
-         -DPER_THREAD_DEFAULT_STREAM=$ENABLE_PTDS \
+         -DCUDF_USE_PER_THREAD_DEFAULT_STREAM=$ENABLE_PTDS \
```

Comment:
Seems like we should update java/src/main/native/CMakeLists.txt to support the new name, as it was intended to keep in sync with the libcudf option.  Can be a followup, but would be nice to fix before 22.06 releases so it's not in an inconsistent state.

Reply:
I will make this change.